### PR TITLE
update db schemes

### PR DESCRIPTION
**Pull request by BobriK502**

### CODE REVIEW

**Review by synthetiquely — APPROVED** [review]

File: src/repositories/tempAdRepository.ts [hunk 1]
```diff
@@ -6,57 +6,52 @@ import { TempAd } from '../db/';
...
+          modelId: doc.modelId,
+          price: doc.price,
+          sourceName: doc.sourceName,
+          sourceUrl: doc.sourceUrl,
+          year: doc.year
+        });
+        await newAd.save();
+      }
+    })
+    .then(async () => await dropCollection());
```
Comment (AntonNarkevich) [inline src/repositories/tempAdRepository.ts]: Why are we combining .then's and awaits? Please get rid of then here.

File: src/utils/parserUtils.ts [hunk 2]
```diff
@@ -8,6 +8,18 @@ import { getMarkByName } from '../services/markService';
 import { updateMarks } from '../services/markService';
 import { getModelByName, saveNewModel } from '../services/modelService';
 
+export const transformOnlinerDate = (onlinerDate: string) => {
+  let date = onlinerDate.substring(0, 10);
```
Comment (AntonNarkevich) [inline src/utils/parserUtils.ts]: Please use moment.js here.

**Review by AntonNarkevich — CHANGES_REQUESTED** [review]
Pretty much looks good. Please fix 2 moments I commented and let's merge.

File: src/utils/parserUtils.ts (hunk 2 above)
Comment (synthetiquely) [reply src/utils/parserUtils.ts]: Should we really add an external library for only one small piece of code?

**Pull request merged**